omnipaxos_core/src/storage.rs

```diff
@@ -560,9 +557,14 @@ where
         self.storage.get_entries(from_sfx_idx, to_sfx_idx)
     }
 
+    /// Update the accepted index in the state cache
+    fn update_accepted_idx(&mut self) {
+        self.state_cache.set_accepted_idx(self.get_real_log_len() + self.get_compacted_idx());
+    }
+
```

Comment:
Remove this function. It is more clear to directly do `self.state_cache.set_accepted_idx()` and give the compacted idx manually. 

In general, please remove all these wrapper functions with a single line that just `update` or `read` from the cache. You might need to keep some of them that are being called from `SequencePaxos`, but try removing the rest. They make it confusing since they are inside InternalStorage, but are actually updating and reading from the cache.